README.md

```diff
@@ -27,6 +27,7 @@ The Bamboo.MandrillAdapter and Bamboo.SendgridAdapter **are being used in produc
 and have had no issues. It's also pretty simple to [create your own adapter]. Feel free
 to open an issue or a PR if you'd like to add a new adapter to the list.
 
+* `Bamboo.MailgunAdapter` - Ships with Bamboo. Thanks to @princemaple.
```

Comment:
If someone did a large feature almost entirely by themselves, then I like to do it. It’s a nice sign and I think that it will encourage more contribution. I love it when I contribute and the maintainers mention it :)
> On Apr 29, 2016, at 4:28 PM, Tute Costa <notifications@github.com> wrote:
> 
> In README.md <https://github.com/thoughtbot/bamboo/pull/140#discussion_r61639989>:
> 
> > @@ -27,6 +27,7 @@ The Bamboo.MandrillAdapter and Bamboo.SendgridAdapter **are being used in produc
> >  and have had no issues. It's also pretty simple to [create your own adapter]. Feel free
> >  to open an issue or a PR if you'd like to add a new adapter to the list.
> >  
> > +* `Bamboo.MailgunAdapter` - Ships with Bamboo. Thanks to @princemaple.
> More general: should we mention individual contributors in the project's README?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/thoughtbot/bamboo/pull/140/files/106b4a7277374f2306b2bb4aaddcd8e876ea2887#r61639989>